pkgs/tools/networking/tayga/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, autoconf, automake }:
+
+stdenv.mkDerivation rec {
+  version = "0.9.2";
+  name = "tayga-${version}";
+
+  src = fetchurl {
+    url= "http://www.litech.org/tayga/tayga-${version}.tar.bz2";
+    sha256 = "1700y121lhvpna49bjpssb7jq1abj9qw5wxgjn8gzp6jm4kpj7rb";
+  };
+
+  nativeBuildInputs = [ autoconf automake ];
+
+  meta = with stdenv.lib; {
+    description = "Userland stateless NAT64 daemon";
+    longDescription = ''
+      TAYGA is an out-of-kernel stateless NAT64 implementation
+      for Linux that uses the TUN driver to exchange IPv4 and
+      IPv6 packets with the kernel.
+      It is intended to provide production-quality NAT64 service
+      for networks where dedicated NAT64 hardware would be overkill.
+    '';
+    homepage = http://www.litech.org/tayga;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.0x4A6F ];
+    platforms = platforms.unix;
```

Comment:
After reading the homepage, this looks linux specific to me. I may be wrong, though.

Reply:
It is also in FreeBSD ports as [net/tayga](https://www.freshports.org/net/tayga).

Reply:
It does not build on macOS though.

Reply:
So back to linux only for now.